CONTRIBUTING.md

```diff
@@ -14,21 +14,20 @@ cd mkdocstrings
 make setup
 ```
 
-!!! note
-    If it fails for some reason,
-    you'll need to install
-    [PDM](https://github.com/pdm-project/pdm)
-    manually.
-
-    You can install it with:
-
-    ```bash
-    python3 -m pip install --user pipx
-    pipx install pdm
-    ```
-
-    Now you can try running `make setup` again,
-    or simply `pdm install`.
+> NOTE: If it fails for some reason,
+> you'll need to install
+> [PDM](https://github.com/pdm-project/pdm)
+> manually.
+> 
+> You can install it with:
+> 
+> ```bash
+> python3 -m pip install --user pipx
+> pipx install pdm
+> ```
+> 
+> Now you can try running `make setup` again,
+> or simply `pdm install`.
```

Comment:
Switched to callouts :heart: 
This one was important to change since CONTRIBUTING is often read on GitHub directly.
I did not switch all admonitions to callouts, because the markup tends to be heavy, but I'm not opposed to a full switch later.